### PR TITLE
metazoan archives block updated and few typos fixed

### PR DESCRIPTION
**Pull request by vsitnik**

### CODE REVIEW

**Review by ens-st3 — COMMENTED** [review]
Why remove eg-37 ?

File: metazoa/species/Actinia/Actinia_equina/Actinia_equina_GCA_011057435.1_assembly.md [hunk 1]
```diff
@@ -2,7 +2,8 @@ Assembly
 --------
 
 The *Actinia equina* (_AE1_ strain) genome was sequenced at the [Centre for Genomic Research, University of Liverpool](https://www.liverpool.ac.uk/genomic-research/).
-Assembly  was undertaken using three separate assembly methods: CANU v1.7, SMARTdenovo or WTDBG [1] and submitted by the
-[Liverpool John Moores University](http://www.ljmu.ac.uk).
+
+Assembly  was undertaken using three separate assembly methods: CANU v1.7, SMARTdenovo, WTDBG and submitted by the
```
Comment (ens-st3) [inline metazoa/species/Actinia/Actinia_equina/Actinia_equina_GCA_011057435.1_assembly.md:6]: Extra whitespace between 'Assembly' and 'was'

Comment (vsitnik) [reply metazoa/species/Actinia/Actinia_equina/Actinia_equina_GCA_011057435.1_assembly.md:6]: Thanks for spotting, and I'll return eg-37.

Comment (vsitnik) [reply metazoa/species/Actinia/Actinia_equina/Actinia_equina_GCA_011057435.1_assembly.md]: eg-37 is back, typo fixed

**Pull request merged**